docs/bosh/README.md

```diff
@@ -119,6 +119,9 @@ Now you have the infrastructure ready to deploy a BOSH director.
 1. SSH to the bastion VM you created in the previous step. You can use Cloud Shell to SSH to the bastion, or you can connect from any workstation with `gcloud` installed. All SSH commands after this should be run from the bastion VM.
 
   ```
+  gcloud auth login
```

Comment:
These instructions shouldn't be required if using Cloud Shell. Can you break these out with a note to that effect?